lib/administrate/base_dashboard.rb

```diff
@@ -62,11 +62,18 @@ def specific_form_attributes_for(action)
       self.class.const_get(cname) if self.class.const_defined?(cname)
     end
 
-    def permitted_attributes
-      form_attributes.map do |attr|
+    def permitted_attributes(action = nil)
+      action =
+        case action
+        when "update" then "edit"
+        when "create" then "new"
+        else action
+        end
```

Comment:
This code also exists in `lib/administrate/page/form.rb` where it's also used ahead of calling `form_attributes`. What do you think of removing it from both places and putting it at the start of `form_attributes` instead?

Reply:
done